src/components/table-of-contents/_macro.spec.js

```diff
@@ -55,6 +55,43 @@ const EXAMPLE_TABLE_OF_CONTENTS_MULTIPLE = {
     ],
 };
 
+const EXAMPLE_TABLE_OF_CONTENTS_RELATED_LINKS_BUTTON = {
+    title: 'Contents',
+    itemsList: [
+        {
+            url: '#overview',
+            text: 'Overview',
+        },
+        {
+            url: '#who-should-take-part-and-why',
+            text: 'Who should take part and why',
+        },
+    ],
+    relatedLinksTitle: 'Related publications',
+    relatedLinksList: [
+        {
+            itemsList: [
+                {
+                    url: '#0',
+                    text: 'Example publication title',
+                },
+                {
+                    url: '#0',
+                    text: 'Example publication title 2',
+                },
+                {
+                    url: '#0',
+                    text: 'Example publication title 3',
+                },
+            ],
+        },
+    ],
+    button: {
+        text: 'Save or print this page',
+        variants: ['print', 'small', 'secondary'],
```

Comment:
I think we are likely to always want these set so maybe we should hardcode these in the macro